webpack/test_setup.js

```diff
@@ -1,4 +1,13 @@
 jest.mock('jed');
+jest.mock(
+  './assets/javascripts/react_app/ReactApp/Context/ForemanContext',
```

Comment:
I am wondering if we should create a convention around the mocking.

For example, create a file called `./assets/javascripts/react_app/ReactApp/Context/ForemanContext.mocks.js` and make jest to use this mock automatically.

Reply:
AFAIK jest has a convention with a `__mocks__` folder.

Reply:
@sharvit Yeah I think Jest will use its own implementation for `jest.mock` unless you provide a mock in `__mocks__`.

Reply:
I agree we need a best practice for mocking, but it's not related to this scope